src/find-paginated.ts

```diff
@@ -1,92 +1,78 @@
-import { Collection } from 'mongodb'
+import { Collection } from "mongodb";
 
-import { BaseDocument, Projection, Query, Sort } from './types'
-import {
-  buildCursor,
-  buildQueryFromCursor,
-  encodeCursor,
-  normalizeDirectionParams,
-} from './utils'
+import { BaseDocument, Projection, Query, Sort } from "./types";
+import { buildCursor, buildQueryFromCursor, encodeCursor, normalizeDirectionParams } from "./utils";
 
-export type FindPaginatedParams = {
-  first?: number | null
-  after?: string | null
-  last?: number | null
-  before?: string | null
-  query?: Query
-  sort?: Sort
-  projection?: Projection
+export interface FindPaginatedParams {
+    first?: number | null;
+    after?: string | null;
+    last?: number | null;
+    before?: string | null;
+    query?: Query;
+    sort?: Sort;
+    projection?: Projection;
 }
 
-export type FindPaginatedResult<TDocument> = {
-  edges: Array<{ cursor: string; node: TDocument }>
-  pageInfo: {
-    startCursor: string | null
-    endCursor: string | null
-    hasPreviousPage: boolean
-    hasNextPage: boolean
-  }
+export interface FindPaginatedResult<TDocument> {
```

Comment:
it will be nice to keep the interface in separated file then code (logic) https://github.com/Newforma/mongo-cursor-pagination-alt/pull/5/files#diff-31bafb7ca53774818d9af6cc32250345b848fd30b03f978156577f8c2f4a965fR26

Reply:
This entire repo is forked from the public GitHub repository. We did not change it in years. It works, so I would like not to change the code where it is possible unless it is needed to improve its functionality.

Reply:
can you update the PR description what changes were made (refactoring) with this PR, I don't see only changes with github npm registry

Reply:
Done